features/api/ideas_workshop_ideas.feature

```diff
@@ -823,17 +823,12 @@ Feature:
   Scenario: As a logged-in user I can not publish an idea that has another status than PENDING at the moment of execution
     Given I am logged as "jacques.picard@en-marche.fr"
     When I send a "PUT" request to "/api/ideas/4/publish"
-    Then the response status code should be 403
+    Then the response status code should be 400
 
   Scenario: As a logged-in user I get errors when I publish my idea with few data
```

Comment:
Please, change also this scenario description.

Reply:
done